questions/includes/fedora/coding.yml

```diff
@@ -2,30 +2,38 @@ tree:
   children:
     - title: Python
       subtitle: a dynamic language for rapid development
-      segue1: So, you enjoy traceback-driven development?
+      segue1: So, you enjoy traceback-driven development?  What could go wrong?
       segue2: Are you interested in hacking on...
       children: coding/python.yml
 
     - title: C
       subtitle: close to the metal
-      href: https://lkml.org
+      segue1: So, you think OOP is for hipsters?  Let me tell you, back in my day...
+      segue2: How about working on...
+      children: coding/c.yml
 
     - title: Haskell
       subtitle: a nondynamic language for rapid development
-      segue1: Yay, types!
+      segue1: So, programming without side-effects is what you're all about?  Sounds useful!
       segue2: How about...
       children: coding/haskell.yml
 
     - title: Java
       subtitle: a nondynamic language for nonrapid development
+      segue1: So you're a believer in <code>AbstractMethodFactoryBean</code>s?  Straightforward enough...
+      segue2: How about...
       children: coding/java.yml
 
     - title: Scala
       subtitle: a nondynamic language for semirapid development
+      segue1: So, just OOP or functional languages wouldn't do it?  You had to have both, huh?
+      segue2: How about working on...
       children: coding/scala.yml
 
     - title: JavaScript
       subtitle: a dynamic web-oriented language for backend and frontend
+      segue1: So, you think having native integers is over-rated?  What could go wrong?
```

Comment:
> How about something like: "I don't [Object object] to you spending NaN hours on this!" :P

Yes to this.  But I'll save it because..

> could we make the segues optionally be a list and have it pick one randomly?

Totally.  Let's make it a separate PR and go nuts.